public/plugin.ts

```diff
@@ -66,6 +69,7 @@ export class MlCommonsPluginPlugin
           label: i18n.translate('MLCommonsDashboards.Category.MachineLearning.label', {
             defaultMessage: 'Machine learning',
           }),
+          order: 9999,
```

Comment:
how it looks like in left nav?

Reply:
That should be at the bottom of the left navigation. I've update the screenshot.